api/tests/functional-tests/backend/metrics/test_detection.py

```diff
@@ -2292,11 +2292,11 @@ def test__convert_annotations_to_common_type(db: Session):
     mask[ymin:ymax, xmin:xmax] = True
 
     pts = [
-        (xmin, ymin),
-        (xmin, ymax),
-        (xmax, ymax),
-        (xmax, ymin),
-        (xmin, ymin),
+        (float(xmin), float(ymin)),
```

Comment:
why are these types changing if you're just updating the schema for LLM-related metrics?

Reply:
This is related to updating to the latest pre-commit version, for some reason the handling of typing `tuple[int | float, int | float]` has changed.